scripts/quests/otherAreas/The_Basics.lua

```diff
@@ -0,0 +1,196 @@
+-----------------------------------
+-- The Basics
+-----------------------------------
+-- Zone,    NPC,          POS
+-- Mhaura,  Rycharde,     !pos
+-- Selbina, Valgeir,      !pos
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require("scripts/globals/titles")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local mhauraID  = require("scripts/zones/Mhaura/IDs")
+local selbinaID = require("scripts/zones/Selbina/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.THE_BASICS)
+-----------------------------------
+
+quest.reward =
+{
+    -- exp     = 2000,
+    fame    = 120,
+    item    = xi.items.TEA_SET,
+    title   = xi.title.FIVESTAR_PURVEYOR,
+}
+
+quest.sections =
+{
+    -- Section: Quest is available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and player:getFameLevel(WINDURST) > 4 and
+                player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.THE_CLUE) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.MHAURA] =
+        {
+            ['Rycharde'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getCharVar("TheClueCompDay") + 7 < VanadielDayOfTheYear() or player:getCharVar("TheClueCompYear") < VanadielYear() then
+                        return quest:event(94) -- Quest starting event.
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [94] = function(player, csid, option, npc)
+                    if option == 85 then -- Accept quest option.
+                        player:setCharVar("TheClueCompDay", 0)  -- Delete previous quest (The clue) variables.
+                        player:setCharVar("TheClueCompYear", 0) -- Delete previous quest (The clue) variables.
+                        player:addKeyItem(xi.ki.MHAURAN_COUSCOUS) -- Give Key Item to player.
```

Comment:
Can use nocUtil.giveKeyItem() and drop the message special line possibly even the locals pointing to the id page